config/settings/base.py

```diff
@@ -583,6 +583,19 @@
     ),
 )
 
+USER_S3_BUCKET = env("USER_S3_BUCKET", default="")
+USER_S3_REGION = env("USER_S3_REGION_CODE", default=BUCKET_REGION)
+USER_S3_KEY = env("USER_S3_KEY", default=BUCKET_KEY)
+USER_S3_SECRET = env("USER_S3_SECRET", default=BUCKET_SECRET)
+USER_S3_BUCKET_ENDPOINT = env("USER_S3_BUCKET_ENDPOINT", default=BUCKET_ENDPOINT)
+USER_S3_BUCKET_EXTERNAL_ENDPOINT = env(
+    "USER_S3_BUCKET_EXTERNAL_ENDPOINT",
+    default= BUCKET_EXTERNAL_ENDPOINT
+    if BUCKET_ENDPOINT
+    else USER_S3_BUCKET_ENDPOINT,
+)
+
+
```

Comment:
Lets re-use the facility bucket itself for user avatars. cc: @sainak

Reply:
@gigincg @vigneshhari

Reply:
can someone please confirm which bucket to use?